To: vim_dev@googlegroups.com
Subject: Patch 7.3.391
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.391
Problem:    Can't check if the XPM_W32 feature is enabled.
Solution:   Add xpm_w32 to the list of features. (kat)
Files:	    src/eval.c


*** ../vim-7.3.390/src/eval.c	2011-12-14 15:32:44.000000000 +0100
--- src/eval.c	2012-01-04 14:33:22.000000000 +0100
***************
*** 12260,12265 ****
--- 12260,12268 ----
  #ifdef FEAT_XFONTSET
  	"xfontset",
  #endif
+ #ifdef FEAT_XPM_W32
+ 	"xpm_w32",
+ #endif
  #ifdef USE_XSMP
  	"xsmp",
  #endif
*** ../vim-7.3.390/src/version.c	2011-12-30 15:01:55.000000000 +0100
--- src/version.c	2012-01-04 14:34:31.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     391,
  /**/

-- 
ROBIN:  The what?
ARTHUR: The Holy Hand Grenade of Antioch.  'Tis one of the sacred relics
        Brother Maynard always carries with him.
ALL:    Yes. Of course.
ARTHUR: (shouting) Bring up the Holy Hand Grenade!
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
